src/prisma/schema.prisma

```diff
@@ -424,6 +425,10 @@ model Page {
   @@index([proposalId])
   @@index([spaceId])
   @@index([title])
+  @@index([path])
+  // https://www.prisma.io/docs/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql
+  // https://www.postgresql.org/docs/current/indexes-types.html#INDEXES-TYPES-GIN
+  @@index([additionalPaths], type: Gin)
```

Comment:
these should probably be a composite index, since we'd always look them up together. i don't know if that's possible with the Gin type

Reply:
Updated to composite index, and defaulted to Btree